### ISSUE TITLE
cdv:platform defaults to remove.

### ISSUE THREAD

**Issue by kitsunde** [isleofcode/ember-cordova#148]
It's a bit surprising since `cordova platform` is a list, but here it's destructive.

```bash
$ ember cordova:platform android
Preparing to remove platform android
```

**Comment by alexblom** [comment]
Agreed that this should not default to remove. I'll probably be looking into this earlier next week.

**Comment by alexblom** [comment]
Fixed in 0.3.2. We have a meta objective to support more of the cdv footprint, but this behaviour should have been throwing you a warning to include an 'add' or 'remove' flag.
